### DISCUSSION TITLE
product.download()

### DISCUSSION THREAD

**Discussion by tiqvahub** [CS-SI/eodag#901] [Q&A]
Hi all,

While trying to download the cell S1A_IW* the server never responds until it sends the error **ReadTimeoutError: HTTPSConnectionPool(host='peps.cnes.fr', port=443): Read timed out. (read timeout=60)**

**[Retry #1] Waiting 119.999979s until next download try ordered product (retry every 2' for 20')**

Bellow the piece of code:

product = products[1]
product_path = product.downlaod()
product_path

I would appreciate any help.

Thank you for your time in advance.

**Comment by sbrunato** [comment]
hello @tiqvahub , do you still have this error ? `peps` may have been in maintenance when you tried to download these products
